frontend/src/js/route-config/route-config.js

```diff
@@ -443,6 +443,13 @@
             title: "Accept challenge invitation"
         };
 
+        var get_submission_related_files = {
+            name: "get-submission-related-files",
+            url: "/:bucket/*key",
```

Comment:
Please change the URL to something like `evalai.cloudcv.org/web/submission/files?bucket=<blah>&key=<blah>`

Reply:
@deshraj I think we should not let users know what the bucket is or key is. We should parse it on our own in the controller.

How about this url -- `evalai.cloudcv.org/web/submission/files/<bucket_name>/<key>`